PluralKit.Bot/Handlers/ReactionAdded.cs

```diff
@@ -1,3 +1,5 @@
+using System.Diagnostics.Tracing;
```

Comment:
This isn't needed, betting Visual Studio added it automatically for some reason?